docs/pages/docs/ci/github-actions.mdx

```diff
@@ -127,9 +122,14 @@ Create file called `.github/workflows/ci.yml` in your repository with the follow
 
           steps:
             - name: Check out code
-              uses: actions/checkout@v2
+              uses: actions/checkout@v3
+
+            - name: Setup Cache
+              uses: actions/cache@v3
               with:
-                fetch-depth: 2
+                path: |
+                  node_modules/.cache/turbo
```

Comment:
The contents of this path are effectively append-only. Caching it, even with properly-targeted cache keys, does not prevent unbounded growth. (Cache key results in a hit, part of the `turbo run ___` generates a new cache, the later `save` step adds to it. Rinse/repeat ad nauseam and GitHub cache is exhausted.